.github/workflows/test-go.yml

```diff
@@ -45,8 +45,8 @@ jobs:
     concurrency:
       group: kogito_serverless_operator_pr-integration-${{ matrix.container-engine }}-${{ github.head_ref }}
       cancel-in-progress: true
-    # should be set back to 120 once kogito-swf-builder 1.33 is released
-    timeout-minutes: 700
+    # should be set back to 120 here and stable and not kogito-swf-builder-nightly once kogito-swf-builder 1.33 is released
```

Comment:
what do you mean with 'and not kogito-swf-builder-nightly'?

Reply:
Nightly is the updated version by @radtriste  to reduce the time to the original 50 sec, when will be realsed the 1.33 that version will be the released version, now the nighlty is the image with the fix from Tristan

Reply:
asked because the message is confusing. not sure if we want to have it there.